client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -17,6 +17,7 @@ type TracksGoalsSelectEventProperties = {
 	combo: string;
 	total: number;
 	write?: number;
+	'paid-subscribers'?: number;
```

Comment:
Had to add this here to satisfy TypeScript, even if we aren't dealing with "goals" step, but we're loading from the same "goals" list.